master/reference/calicoctl/commands/node/run.md

```diff
@@ -0,0 +1,137 @@
+
+```
+Usage:
+  calicoctl node run [--ip=<IP>] [--ip6=<IP6>] [--as=<AS_NUM>]
+                     [--name=<NAME>]
+                     [--log-dir=<LOG_DIR>]
+                     [--node-image=<DOCKER_IMAGE_NAME>]
+                     [--backend=(bird|gobgp|none)]
+                     [--config=<CONFIG>]
+                     [--no-default-ippools]
+                     [--dryrun]
+
+Options:
+  -h --help                Show this screen.
+     --as=<AS_NUM>         The default AS number for this node.  If this is not
+                           specified, the node will use the global AS number
+                           (see 'calicoctl config' for details).
+     --name=<NAME>         The name of the Calico node.  If this is not
+                           supplied it defaults to the host name.
+     --ip=<IP>             The local management address to use.  If this is not
+                           specified, the node will attempt to auto-discover
+                           the local IP address to use - however, it is
+                           recommended to specify the required address to use.
+     --ip6=<IP6>           The local IPv6 management address to use.  If this
+                           is not specified, the node will not route IPv6.
+     --log-dir=<LOG_DIR>   The directory containing Calico logs.
+                           [default: /var/log/calico]
+     --node-image=<DOCKER_IMAGE_NAME>
+                           Docker image to use for Calico's
+                           per-node container.
+                           [default: calico/node:%s]
+     --backend=(bird|gobgp|none)
+                           Specify which networking backend to use.  When set
+                           to "none", Calico node runs in policy only mode.
+                           The option to run with gobgp is currently
+                           experimental.
+                           [default: bird]
+     --dryrun              Output the appropriate Docker command, without
+                           starting the container.
+     --no-default-ippools  Do not create default pools upon startup.
+                           Default IP pools will be created if this is not set
+                           and there are no pre-existing Calico IP pools.
+  -c --config=<CONFIG>     Filename containing connection configuration in
+                           YAML or JSON format.
+                           [default: /etc/calico/calicoctl.cfg]
+
+Description:
+  This command is used to start a Calico node container instance.  The
+  Calico node is used to provide Calico networking on your compute host.
+
+  This command is used to quickly start the Calico node container using Docker
+  and by running with the --dryrun option can display the appropriate Docker
+  command without actually running the command - this is useful if you intend
+  to deploy Calico and therefore should include in your system startup
+  configuraiton (e.g. systemd).
```

Comment:
Delete whole para?  I don't see anything helpful here.

(Or if not, fix 'configuration' typo.)